qdl.c

```diff
@@ -213,23 +224,25 @@ int main(int argc, char **argv)
 		}
 	} while (++optind < argc);
 
-	ret = qdl_open(&qdl, serial);
+	ret = qdl_open(qdl, serial);
```

Comment:
This and the below renames from qdl_ to usb_ prefix is a logically separate change from the introduction of the simulation mechanism. Please perform such trivial adjustments in a separate patch (in the same PR).

Reply:
I've split on two commits